cmd/egress/cmd.go

```diff
@@ -144,6 +144,11 @@ are set correctly before execution.
 
 				awsVerifier.Logger.Warn(context.TODO(), "Using region: %s", config.region)
 
+				if vei.Proxy.NoTls && vei.Proxy.Cacert != "" {
+					awsVerifier.Logger.Warn(context.TODO(), "CaCert: %s, will not be used since --NoTls flag was used", vei.Proxy.Cacert)
```

Comment:
A more succinct log like this can also be pulled up with my other suggestion

```
awsVerifier.Logger.Warn(context.TODO(), "--cacert value ignored since --no-tls was passed")
```

Reply:
done